k8s/namespaces/nfdiv/nfdiv-cron-migrate-bulk-cases/perftest-00.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-migrate-bulk-cases
+spec:
+  releaseName: nfdiv-cron-migrate-bulk-cases
+  values:
+    job:
+      suspend: true
+      environment:
+        NOTIFY_TEMPLATE_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.perftest.platform.hmcts.net/
+        NOTIFY_TEMPLATE_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.perftest.platform.hmcts.net/
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DIVORCE_URL: https://nfdiv-apply-for-divorce.perftest.platform.hmcts.net/applicant2
+        NOTIFY_TEMPLATE_APPLICANT2_SIGN_IN_DISSOLUTION_URL: https://nfdiv-end-civil-partnership.perftest.platform.hmcts.net/applicant2
+        S2S_URL: http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal
+        FEE_API_URL: http://fees-register-api-aat.service.core-compute-perftest.internal
+        IDAM_API_REDIRECT_URL: https://div-pfe-perftest.service.core-compute-perftest.internal/authenticated
+        IDAM_API_BASEURL: "https://idam-api.perftest.platform.hmcts.net"
+        CASE_DATA_STORE_BASEURL: "http://ccd-data-store-api-perftest.service.core-compute-perftest.internal"
```

Comment:
don't think we need those urls as they will be replaced based on environment

Reply:
ok cool, will remove 👍